codespell_lib/data/dictionary.txt

```diff
@@ -7795,11 +7880,27 @@ bridg->bridge
 brievely->briefly
 brievety->brevity
 brigde->bridge
```

Comment:
Tried to find occurrences of `brigade` typos, without much success.

Reply:
There are occurrences of `fire brigde`->`fire brigade`. I will add this.